projects/metis/src/app/dataset/workflow/workflow.component.ts

```diff
@@ -382,9 +402,11 @@ export class WorkflowComponent extends SubscriptionManager implements OnInit {
     }
     // parameters for link-checking
     if (enabledPluginMetadata.pluginType === 'LINK_CHECKING') {
-      this.workflowForm.controls.performSampling.setValue(
-        enabledPluginMetadata.performSampling ? 'true' : 'false'
-      );
+      let value = 'false';
+      if (enabledPluginMetadata.performSampling) {
+        value = 'true';
+      }
```

Comment:
I think you can replace these four lines by something like this?
```
let value = String(enabledPluginMetadata.performSampling);
```

Reply:
Yes